domains/yunexiz.json

```diff
@@ -4,6 +4,6 @@
         "email": "yunexiz@aol.com"
     },
     "record": {
-        "A": ["45.137.70.68"]
+        "A": ["109.71.252.37"]
```

Comment:
I cannot reach your website (500 internal server error)

Reply:
the ip is not a single server ip, (if thats what you were thinking) its a proxy

Reply:
for the website its bali.weissx.host:14104 but im trying to proxy yunexiz.is-a.dev on it

Reply:
try this link: 109.71.252.37:14104

Reply:
try this link: 109.71.252.37:14104